sdk/search/Azure.Search.Documents/src/AzureSearchDocumentsEventSource.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+using Azure.Core.Diagnostics;
+
+namespace Azure.Search.Documents
+{
+    [EventSource(Name = EventSourceName)]
+    internal sealed class AzureSearchDocumentsEventSource : EventSource
```

Comment:
Should we specialize this to the buffered sender?  I'm not aware of any plans to log anything custom for the rest of Search.

Reply:
What would be the benefit of narrowing the scope of the event source? It is a singleton available at a service level and IMO we can use it at different places without creating several of these (each focused on a small area).

Reply:
Also, the guidance is 1 ES per assembly.